packages/shared/src/components/image/VideoImage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import PlayIcon from '../icons/Play';
+import { IconSize } from '../Icon';
+import { ImageProps, Image } from './Image';
+
+export interface VideoImageProps extends ImageProps {
+  size?: IconSize;
+  wrapperClassName?: string;
+}
+
+const VideoImage = ({
+  size = IconSize.XXXLarge,
+  wrapperClassName,
+  ...props
+}: VideoImageProps): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        wrapperClassName,
+        'flex relative justify-center items-center w-full h-auto rounded-xl',
+      )}
+    >
+      <>
+        <span className="absolute w-full h-full rounded-xl bg-overlay-tertiary-black" />
+        <PlayIcon
+          secondary
+          size={size}
+          data-testid="playIconVideoPost"
+          className="absolute"
+        />
+      </>
```

Comment:
I think we don't need the Fragment anymore. Also we should not be using the tailwind defined rounded classes.
```suggestion
      <span className="absolute w-full h-full rounded-12 bg-overlay-tertiary-black" />
      <PlayIcon
        secondary
        size={size}
        data-testid="playIconVideoPost"
        className="absolute"
      />
```

Reply:
Ah yes, missed it.

Reply:
As for the rounded classes, I saw the xl used in Card components so went with the same class, wasn't aware we shouldn't use them

Reply:
Yep, legacy stuff. It is easier for us to understand as the classes with explicit numbers came from our design system.